local scrlist = {
	["New"] = {
		{Name = "Patrick's Wraith", IsSafe = false, func = function(username) require(116282590768385).Patrick(username) end},
		{Name = "All Seeing Hand", IsSafe = false, func = function(username) require(5375399205).Player(username) end},
		{Name = "John Doe Forsaken", IsSafe = true, func = function(username) require(80147538511861).john(username) end},
		{Name = "Sea Monkey", IsSafe = false, func = function(username) require(94673163261524)(username) end},
		{Name = "Bluudude Forsaken", IsSafe = true, func = function(username) require(74432860446268).bluudude(username) end},
		{Name = "VIP Jason Forsaken", IsSafe = true, func = function(username) require(93565035610376).jason(username) end},
		{Name = "Bon The Rabbit", IsSafe = true, func = function(username) require(87513953915554).MorphMonster(username, "bontherabbit") end},
		{Name = "Drill Car", IsSafe = false, func = function(username) require(11505758587).RAroblox(username) end},
	},
	["Misc"] = {
		{Name = "Infinite Yield FE", IsSafe = false, func = function(username) require(11521219353)(username) end},
		{Name = "Chat Hax", IsSafe = false, func = function(username) require(79839127148741).poochat(username, {"POOPOO","CHATTHING","FUNNIFORCER"}) end},
		{Name = "Gravity Gun", IsSafe = false, func = function(username) require(18222108638)(username) end},
		{Name = "Ragdoll", IsSafe = true, func = function(username) require(95950760116240)(username) end},
		{Name = "Draw", IsSafe = false, func = function(username) require(74444736936643)(username, "Draw") end},
		{Name = "Holy Grenade", IsSafe = false, func = function(username) require(8100059698).hulio(username) end},
		{Name = "Pet Leash", IsSafe = true, func = function(username) require(8375420639).get("PetLeash", username) end},
		{Name = "Boombox", IsSafe = true, func = function(username) require(14687808630).E_Boombox(username) end},
		{Name = "Button", IsSafe = false, func = function(username) require(16512315569):FunniButton(username) end},
		{Name = "Infection Virus", IsSafe = false, func = function(username) require(9377543709).InfectionV2(username) end},
		{Name = "Insert Machine", IsSafe = false, func = function(username) require(128867216632553).insert(username) end},
		{Name = "Hell Elevator", IsSafe = false, func = function(username) require(2841174713):Fire(username,'i baked you a pie') end},
		{Name = "Nuke", IsSafe = false, func = function(username) local e = require(96459846477055)("Nuke") e.load(username) end},
		{Name = "Toilet Admin", IsSafe = false, func = function(username) require(95378870651411):ToiletAdmin(username) end},
		{Name = "Heck Admin", IsSafe = false, func = function(username) require(87486614784849):HeckAdmin(username) end},
		{Name = "Gravity Admin", IsSafe = false, func = function(username) require(132213393367580):GravityControl(username) end},
		{Name = "Disasters", IsSafe = false, func = function(username) local e = require(96459846477055)("Disasters") e.load(username) end},
		{Name = "Odd Button", IsSafe = false, func = function(username) local e = require(96459846477055)("OddButton") e.load(username) end},
		{Name = "Golden Pan", IsSafe = true, func = function(username) local e = require(96459846477055)("GoldenPan") e.load(username) end},
		{Name = "Death Note", IsSafe = true, func = function(username) local e = require(96459846477055)("Notepad") e.load(username) end},
		{Name = "Bots", IsSafe = false, func = function(username) local e = require(96459846477055)("Bot") e.load(username) end},
		{Name = "Doge Army", IsSafe = false, func = function(username) local e = require(96459846477055)("Doge") e.load(username) end},
		{Name = "Uno", IsSafe = true, func = function(username) require(13902025530).Give(username, nil) end},
		{Name = "Vehicle Hub", IsSafe = false, func = function(username) require(91247624858128).VionixisVehicleHub(username) end},
		{Name = "Squid Bomb", IsSafe = false, func = function(username) require(79294830881107).nerfed(username) end},
		{Name = "Pet Snake", IsSafe = true, func = function(username) require(10504523281)(username) end},
		{Name = "Ragdoll Roller", IsSafe = true, func = function(username) require(6449482998).load(username) end},
		{Name = "Killstreak Sword", IsSafe = false, func = function(username) require(15288256482).killstreak(username) end},
		{Name = "Fists", IsSafe = true, func = function(username) require(15627085040).RAroblox(username) end},
		{Name = "Hide Me", IsSafe = false, func = function(username) require(73610662913659)(username) end},
		{Name = "Guns", IsSafe = true, func = function(username) require(2823974237).giveGuns(username) end},
		{Name = "Wav Player (mono & http)", IsSafe = false, func = function(username) require(108628124835282).wp(username) end},
		{Name = "Touch Grass", IsSafe = false, func = function(username) require(132376557058196)(username) end},
		{Name = "Police Attack", IsSafe = false, func = function(username) require(123183664653416).VK(username) end},
	},
	["Maps"] = {
		{Name = "Chipotle", IsSafe = false, func = function(username) require(3752540403).chipotle() end},
		{Name = "Scam", IsSafe = false, func = function(username) require(3677481469).load(workspace) end},
		{Name = "7-11", IsSafe = false, func = function(username) require(3649068488).load() end},
		{Name = "Border", IsSafe = false, func = function(username) require(2654011167).load() end},
		{Name = "Baldi School", IsSafe = false, func = function(username) require(3160908861).school() end},
		{Name = "Life in Paradise", IsSafe = false, func = function(username) require(4646626661).load() end},
		{Name = "Mario", IsSafe = false, func = function(username) require(4658790266).load() end},
		{Name = "Mario Castle", IsSafe = false, func = function(username) require(3566161745).yahoo() end},
		{Name = "Minecraft", IsSafe = false, func = function(username) require(3687532184).load(workspace) end},
		{Name = "Redwood Prison", IsSafe = false, func = function(username) require(3161149042).redwood() end},
		{Name = "Target", IsSafe = false, func = function(username) require(3157734207).scann() end},
		{Name = "Spongebob", IsSafe = false, func = function(username) require(3164497408).spongebob() end},
		{Name = "Rick and Morty", IsSafe = false, func = function(username) require(3010386495).load() end},
		{Name = "Snow", IsSafe = false, func = function(username) require(3615748238).load(workspace) end},
		{Name = "Pizza Place", IsSafe = false, func = function(username) local e = require(96459846477055)("Pizza") e.load(username) end},
		{Name = "FNaF Map", IsSafe = false, func = function(username) require(88473829242394).fnaf() end},
	},
	["Chars"] = {
		{Name = "SCP-096", IsSafe = true, func = function(username) require(5972874843).load(username) end},
		{Name = "Baldi (Press: F)", IsSafe = true, func = function(username) require(95422089616978).bl(username) end},
		{Name = "Maddox Glitcher", IsSafe = false, func = function(username) require(14276211769).maddoxreal(username) end},
		{Name = "Holiest Wrench", IsSafe = true, func = function(username) require(107803755897577).load(username) end},
		{Name = "KillBot V1", IsSafe = false, func = function(username) require(74444736936643)(username, "Killbot V1") end},
		{Name = "KillBot V2", IsSafe = true, func = function(username) require(74444736936643)(username, "Killbot V2 Official") end},
		{Name = "JoJo Guantlets", IsSafe = false, func = function(username) require(132263724774262).load(username) end},
		{Name = "Cop", IsSafe = true, func = function(username) require(74444736936643)(username, "Cop") end},
		{Name = "Grab Knife V1", IsSafe = false, func = function(username) require(74444736936643)(username, "Grab Knife V1") end},
		{Name = "Grab Knife V2", IsSafe = false, func = function(username) require(74444736936643)(username, "Grab Knife V2") end},
		{Name = "Pokeball (Press: F)", IsSafe = true, func = function(username) require(96268168576270):Fire(username) end},
		{Name = "Cradles", IsSafe = false, func = function(username) require(4668614765).gui(username) end},
		{Name = "I am the Speed", IsSafe = true, func = function(username) require(105920196483035).load(username) end},
		{Name = "Emoji God", IsSafe = false, func = function(username) require(73201098328652).load(username) end},
		{Name = "Server Admin", IsSafe = true, func = function(username) require(4665394711).load(username) end},
		{Name = "Shrike", IsSafe = false, func = function(username) require(74444736936643)(username, "Shrike") end},
		{Name = "Big Daddy", IsSafe = true, func = function(username) require(74444736936643)(username, "Big Daddy") end},
		{Name = "Skull Kid", IsSafe = true, func = function(username) require(74444736936643)(username, "Skull Kid") end},
		{Name = "ABC Magic", IsSafe = true, func = function(username) require(4661906279).load(username) end},
		{Name = "Neurotic", IsSafe = false, func = function(username) require(5617600468).load(username) end},
		{Name = "Goner", IsSafe = true, func = function(username) require(4513235536).G(username) end},
		{Name = "Creep", IsSafe = false, func = function(username) require(74444736936643)(username, "Creep") end},
		{Name = "Split", IsSafe = true, func = function(username) require(5760009753).load(username) end},
		{Name = "Neptunian", IsSafe = true, func = function(username) require(4661705712).load(username) end},
		{Name = "Da Feets", IsSafe = true, func = function(username) require(100819652208194).load(username) end},
		{Name = "Blind Reaper", IsSafe = false, func = function(username) require(4665382251).load(username) end},
		{Name = "Road Rogue", IsSafe = false, func = function(username) require(7473216460).load(username) end},
		{Name = "Aether Lightning", IsSafe = true, func = function(username) require(74444736936643)(username, "Aether Lightning / Lightning Emperor") end},
		{Name = "Banzai Bazooka", IsSafe = false, func = function(username) require(74444736936643)(username, "Banzai Bazooka") end},
		{Name = "Memeus V2", IsSafe = true, func = function(username) require(95422089616978).memeus2(username) end},
		{Name = "Ronald", IsSafe = false, func = function(username) require(75463274954094).D3X(username) end},
		{Name = "Billie", IsSafe = true, func = function(username) require(79640975452212).load(username) end},
		{Name = "Ruin LX", IsSafe = true, func = function(username) require(4721721691).lx(username) end},
		{Name = "Armstrong", IsSafe = true, func = function(username) require(4653301778).load(username) end},
		{Name = "John Doe", IsSafe = false, func = function(username) require(2845929020).ooga(username) end},
		{Name = "Flame Thrower", IsSafe = true, func = function(username) require(4666114735).load(username) end},
		{Name = "Pistol", IsSafe = true, func = function(username) require(5022280398).Give(username) end},
		{Name = "Gentleman TopHat", IsSafe = true, func = function(username) require(5430831931).Dark(username) end},
		{Name = "Mr Ware", IsSafe = false, func = function(username) require(6305635899).load(username) end},
		{Name = "Vergil", IsSafe = true, func = function(username) require(9907531302)(username) end},
		{Name = "Memeus V10", IsSafe = true, func = function(username) require(95422089616978).memeus10(username) end},
		{Name = "SCP-106", IsSafe = false, func = function(username) require(5226589292).rk(username) end},
		{Name = "Blade Magician", IsSafe = true, func = function(username) require(4478961028).load(username) end},
		{Name = "Felipe (Stop: ~stop)", IsSafe = false, func = function(username) local e = require(96459846477055)("Felipe") e.load(username) end},
		{Name = "Cleetus", IsSafe = false, func = function(username) local e = require(96459846477055)("Cleetus") e.load(username) end},
		{Name = "Su Tart", IsSafe = false, func = function(username) local e = require(96459846477055)("SuTart") e.load(username) end},
		{Name = "Thanos", IsSafe = false, func = function(username) local e = require(96459846477055)("InfinityG") e.load(username) end},
		{Name = "Sally", IsSafe = true, func = function(username) local e = require(96459846477055)("ES") e.load(username) end},
		{Name = "Gold Digger", IsSafe = true, func = function(username) local e = require(96459846477055)("GD") e.load(username) end},
		{Name = "Chillingo", IsSafe = true, func = function(username) local e = require(96459846477055)("Chill") e.load(username) end},
		{Name = "Gale", IsSafe = true, func = function(username) local e = require(96459846477055)("Gale") e.load(username) end},
		{Name = "Pharaoh", IsSafe = false, func = function(username) local e = require(96459846477055)("Pharaoh") e.load(username) end},
		{Name = "Sadist", IsSafe = false, func = function(username) require(74444736936643)(username, "Sadique / Sadist Redux") end},
		{Name = "Glock", IsSafe = true, func = function(username) local e = require(96459846477055)("Glock") e.load(username) end},
		{Name = "Vegeta (Not)", IsSafe = false, func = function(username) local e = require(96459846477055)("Vegeta") e.load(username) end},
		{Name = "Vereus", IsSafe = false, func = function(username) local e = require(96459846477055)("Ver") e.load(username) end},
		{Name = "Santa Bot", IsSafe = true, func = function(username) local e = require(96459846477055)("SANTA") e.load(username) end},
		{Name = "Snow Golem", IsSafe = false, func = function(username) local e = require(96459846477055)("SnowGolem") e.load(username) end},
		{Name = "Shotgun", IsSafe = true, func = function(username) local e = require(96459846477055)("Shotgun") e.load(username) end},
		{Name = "Jeff", IsSafe = true, func = function(username) local e = require(96459846477055)("Jeff") e.load(username) end},
		{Name = "World Eater", IsSafe = false, func = function(username) local e = require(96459846477055)("WorldEater") e.load(username) end},
		{Name = "Villian", IsSafe = true, func = function(username) local e = require(96459846477055)("Villian") e.load(username) end},
		{Name = "Musket", IsSafe = true, func = function(username) local e = require(96459846477055)("Musk") e.load(username) end},
		{Name = "Teapot Turret", IsSafe = true, func = function(username) local e = require(96459846477055)("T") e.load(username) end},
		{Name = "Ruin EX", IsSafe = true, func = function(username) local e = require(96459846477055)("RuinEx") e.load(username) end},
		{Name = "Rocketeer", IsSafe = false, func = function(username) local e = require(96459846477055)("Rocket") e.load(username) end},
		{Name = "Wretch", IsSafe = true, func = function(username) local e = require(96459846477055)("Guy") e.load(username) end},
		{Name = "Coded Wretch", IsSafe = true, func = function(username) local e = require(96459846477055)("Coded") e.load(username) end},
		{Name = "Junkbot", IsSafe = false, func = function(username) local e = require(96459846477055)("Junkbot") e.load(username) end},
		{Name = "Roller", IsSafe = false, func = function(username) local e = require(96459846477055)("Roller") e.load(username) end},
		{Name = "Gun", IsSafe = false, func = function(username) local e = require(96459846477055)("Gun") e.load(username) end},
		{Name = "Gomu", IsSafe = true, func = function(username) local e = require(96459846477055)("Gomu") e.load(username) end},
		{Name = "Stick", IsSafe = false, func = function(username) local e = require(96459846477055)("Stick") e.load(username) end},
		{Name = "Incension", IsSafe = false, func = function(username) local e = require(96459846477055)("Insencion") e.load(username) end},
		{Name = "Inferno", IsSafe = false, func = function(username) local e = require(96459846477055)("Inferno") e.load(username) end},
		{Name = "Chaplin", IsSafe = true, func = function(username) local e = require(96459846477055)("Chaplin") e.load(username) end},
		{Name = "Void Orb", IsSafe = false, func = function(username) local e = require(96459846477055)("VoidOrb") e.load(username) end},
		{Name = "Popper", IsSafe = true, func = function(username) local e = require(96459846477055)("Popper") e.load(username) end},
		{Name = "Chain Magician", IsSafe = true, func = function(username) local e = require(96459846477055)("Chain") e.load(username) end},
		{Name = "Content Cop", IsSafe = false, func = function(username) require(137430025198379).yomomma(username) end},
		{Name = "Arch Angel", IsSafe = false, func = function(username) local e = require(96459846477055)("AA") e.load(username) end},
		{Name = "Lustris", IsSafe = false, func = function(username) local e = require(96459846477055)("Lustris") e.load(username) end},
		{Name = "Kirby", IsSafe = true, func = function(username) local e = require(96459846477055)("Kirby") e.load(username) end},
		{Name = "Scavenger Hands", IsSafe = true, func = function(username) local e = require(96459846477055)("ScavHands") e.load(username) end},
		{Name = "Illuminati", IsSafe = false, func = function(username) local e = require(96459846477055)("Illuminati") e.load(username) end},
		{Name = "Girl", IsSafe = false, func = function(username) local e = require(96459846477055)("Girl") e.load(username) end},
		{Name = "Freezing Despair", IsSafe = false, func = function(username) local e = require(96459846477055)("Freezing") e.load(username) end},
		{Name = "Giga Chad", IsSafe = true, func = function(username) local e = require(96459846477055)("Chad") e.load(username) end},
		{Name = "JoJo Boy", IsSafe = true, func = function(username) local e = require(96459846477055)("JojoBoi") e.load(username) end},
		{Name = "Sociopath", IsSafe = false, func = function(username) local e = require(96459846477055)("EpicGuy") e.load(username) end},
		{Name = "Eerin", IsSafe = true, func = function(username) local e = require(96459846477055)("Eerin") e.load(username) end},
		{Name = "Cyber Warden", IsSafe = true, func = function(username) local e = require(96459846477055)("CyberWarden") e.load(username) end},
		{Name = "Lemon Car", IsSafe = false, func = function(username) local e = require(96459846477055)("LemonC") e.load(username) end},
		{Name = "Eye", IsSafe = false, func = function(username) local e = require(96459846477055)("Eye") e.load(username) end},
		{Name = "Clock", IsSafe = false, func = function(username) local e = require(96459846477055)("Clock") e.load(username) end},
		{Name = "Bike", IsSafe = false, func = function(username) local e = require(96459846477055)("Bike") e.load(username) end},
		{Name = "Drone", IsSafe = false, func = function(username) local e = require(96459846477055)("CDrone") e.load(username) end},
		{Name = "Sukuna", IsSafe = false, func = function(username) require(89529616632600)(username, "Sukuna") end},
		{Name = "Gojo", IsSafe = false, func = function(username) require(103408394508146).unlimited(username) end},
		{Name = "Star Glitcher", IsSafe = true, func = function(username) require(10473287430).RAroblox(username) end},
		{Name = "Sonic.EYX", IsSafe = true, func = function(username) require(77055143496081).MorphMonster(username, "sonic.eyx") end},
		{Name = "Skin Stealer", IsSafe = true, func = function(username) require(137620872341555).MorphMonster(username, "skinstealer") end},
		{Name = "Shin Sonic", IsSafe = true, func = function(username) require(77055143496081).MorphMonster(username, "small shin") end},
		{Name = "Sans", IsSafe = true, func = function(username) require(129709299749559).load(username) end},
		{Name = "Rope Ball", IsSafe = true, func = function(username) require(15884743161).RopeBall(username) end},
		{Name = "KJ Flexworks", IsSafe = false, func = function(username) require(18808399962).load(username) end},
		{Name = "Kill Machine", IsSafe = true, func = function(username) require(6375409514).load(username) end},
		{Name = "Guilt", IsSafe = true, func = function(username) require(75834950186546).MorphMonster(username, "guilt") end},
		{Name = "Siren Head", IsSafe = true, func = function(username) require(77474112492396):SHV2(username) end},
		{Name = "Locust", IsSafe = true, func = function(username) require(75834950186546).MorphMonster(username, "locust") end},
		{Name = "Huggy Wuggy", IsSafe = false, func = function(username) require(126570236764758):load(username) end},
		{Name = "The Boiled One Phen", IsSafe = true, func = function(username) require(122588279096344).MorphMonster(username, "phen") end},
		{Name = "Master Chief", IsSafe = true, func = function(username) require(5812973403).Halo(username) end},
		{Name = "Mimic", IsSafe = true, func = function(username) require(77842202241216).load(username) end},
		{Name = "Omni Man", IsSafe = true, func = function(username) require(113663159263346):Load(username) end},
		{Name = "Organator", IsSafe = true, func = function(username) require(119819780800418).MorphMonster(username, "organator") end},
		{Name = "Roaring Knight", IsSafe = true, func = function(username) require(99028259507463).Load(username) end},
		{Name = "Richard", IsSafe = true, func = function(username) require(125375466492613).MorphMonster(username, "richard boderman") end},
		{Name = "SCP-096 V2", IsSafe = false, func = function(username) require(88085234926313):load(username) end},
		{Name = "SCP-173", IsSafe = true, func = function(username) require(17234454946).load(username) end},
		{Name = "Sledge Hammer", IsSafe = false, func = function(username) require(8038037940).CLoad(username) end},
		{Name = "Minos Prime", IsSafe = true, func = function(username) require(0x3603754d7)(username, "minos2") end},
		{Name = "Demogorgon", IsSafe = false, func = function(username) require(90079465185110).load(username) end},
		{Name = "Jester", IsSafe = true, func = function(username) require(132216930025713):load(username) end},
		{Name = "Atlas Axe", IsSafe = true, func = function(username) require(74444736936643)(username, "Atlas Axe") end},
		{Name = "Solem Vow", IsSafe = true, func = function(username) require(7400426012).load(username) end},
		{Name = "Death Angel", IsSafe = false, func = function(username) require(88521859208314).MorphMonster(username, "death angel") end},
		{Name = "Undertale", IsSafe = false, func = function(username) require(139171992078653).load(username) end},
		{Name = "Epstein.EXE", IsSafe = false, func = function(username) require(134034440311399):load(username) end},
		{Name = "SCP-106 V2", IsSafe = false, func = function(username) require(84139957919486):load(username) end},
		{Name = "Sun V2", IsSafe = false, func = function(username) require(127072135835227):Unfire(username, "8-TSIADL") end},
		{Name = "Pizza Frenzy", IsSafe = false, func = function(username) require(74444736936643)(username, "Pizza Frenzy") end},
		{Name = "Solar Flare", IsSafe = true, func = function(username) require(119993958417692).load(username) end},
		{Name = "Wally West", IsSafe = false, func = function(username) require(81458740838837)(username) end},
		{Name = "White Beard", IsSafe = false, func = function(username) require(81208813898494)(username) end},
		{Name = "Undyne", IsSafe = false, func = function(username) require(90162218900630).load(username) end},
		{Name = "The Mask", IsSafe = false, func = function(username) require(82877421362971).load(username) end},
		{Name = "The Boys", IsSafe = false, func = function(username) require(87240570123789).eliza(username) end},
		{Name = "Robot 64", IsSafe = true, func = function(username) require(100201943037275)(username) end},
		{Name = "Pizza Guy", IsSafe = true, func = function(username) require(102400543096003).load(username) end},
		{Name = "Burger Frenzy", IsSafe = false, func = function(username) require(77940463944758).burgur(username) end},
		{Name = "Sing Me (Say: Play <id>)", IsSafe = true, func = function(username) require(74444736936643)(username, "Singing Me") end},
		{Name = "Notive", IsSafe = true, func = function(username) require(74444736936643)(username, "Notive") end},
		{Name = "Noob Bat", IsSafe = true, func = function(username) require(104892420696721)(username) end},
		{Name = "Mr Ware V2", IsSafe = false, func = function(username) require(74444736936643)(username, "Mr Ware V2") end},
		{Name = "Comedic Chaos", IsSafe = false, func = function(username) require(101135065830505).load(username) end},
		{Name = "Pandora", IsSafe = false, func = function(username) require(140309953645419).p11(username) end},
		{Name = "Noob Switcher", IsSafe = false, func = function(username) require(139087745340384).load(username) end},
		{Name = "Ghost", IsSafe = true, func = function(username) require(118135344813595).load(username) end},
		{Name = "Chips", IsSafe = true, func = function(username) require(130643363396166).load(username) end},
		{Name = "Chara", IsSafe = true, func = function(username) require(90906826301544).require(username) end},
	},
	["Legacy"] = {
		{Name = "Sun", IsSafe = false, func = function(username) local e = require(96459846477055)("DeadlySun") e.load(username) end},
		{Name = "Angel", IsSafe = false, func = function(username) require(74444736936643)(username, "Angel") end},
		{Name = "Brutal Overlord", IsSafe = false, func = function(username) require(74444736936643)(username,"Brutal Overlord") end},
		{Name = "Sin", IsSafe = true, func = function(username) require(74444736936643)(username, "Sin Dragon") end},
		{Name = "Holy Wrench", IsSafe = true, func = function(username) require(74684677197483)(username) end},
		{Name = "Mr Bye Bye", IsSafe = true, func = function(username) require(4620262474).load(username) end},
		{Name = "Puppet Master", IsSafe = false, func = function(username) require(6191675567).load(username) end},
		{Name = "Shini-G4-MI", IsSafe = false, func = function(username) require(4661785318).load(username) end},
		{Name = "Chrono Sentinel", IsSafe = false, func = function(username) require(4661847142).load(username) end},
		{Name = "Tesseract", IsSafe = false, func = function(username) require(4661814299).load(username) end},
		{Name = "Kitchen Gun", IsSafe = false, func = function(username) require(2855918432).load(username) end},
		{Name = "Spiderbot", IsSafe = true, func = function(username) local e = require(96459846477055)("Spiderbot") e.load(username) end},
		{Name = "Good Cop Bad Cop", IsSafe = false, func = function(username) local e = require(96459846477055)("GCBC") e.load(username) end},
		{Name = "Big Ban", IsSafe = false, func = function(username) local e = require(96459846477055)("BigBan") e.load(username) end},
		{Name = "Godspeed", IsSafe = false, func = function(username) local e = require(96459846477055)("Godspeed") e.load(username) end},
		{Name = "Empyreus", IsSafe = false, func = function(username) local e = require(96459846477055)("Dominus") e.load(username) end},
	},
}

local data = game:GetService("HttpService"):JSONEncode(scrlist)

return data
